### PR TITLE
fix(theme): Sass deprecation errors

### PR DESCRIPTION
**Pull request by DataM0del**

### CODE REVIEW

File: src-theme/package.json [hunk 1]
```diff
@@ -13,20 +13,20 @@
     "@sveltejs/vite-plugin-svelte": "^3.0.2",
     "@tsconfig/svelte": "^5.0.2",
     "@types/dateformat": "^5.0.2",
-    "sass": "^1.70.0",
-    "svelte": "^4.2.10",
```
Comment (SenkJu) [inline src-theme/package.json:17]: Why did you downgrade Svelte?

Comment (DataM0del) [reply src-theme/package.json:17]: uhh oops

File: src-theme/package.json [hunk 2]
```diff
@@ -13,20 +13,20 @@
...
-    "svelte-check": "^3.6.3",
-    "tslib": "^2.6.2",
-    "typescript": "^5.2.2",
-    "vite": "^5.1.7"
+    "sass": "^1.81.0",
+    "svelte": "^4.0.0",
```
Comment (DataM0del) [inline src-theme/package.json]:
```suggestion
    "svelte": "^4.2.10",
```

File: src-theme/package.json (hunk 1 above)
Comment (DataM0del) [reply src-theme/package.json:17]: Resolved in 3e7c077699b728f76ea92c312b4d289b5ae81621

**Comment by DataM0del** [comment]
We can use the `@mateothegreat/svelte5-router` npm package with Svelte 5. Will update to use that package instead.

**Comment by DataM0del** [comment]
image
lol

**Comment by DataM0del** [comment]
I forgot to commit the `Block_` change :skull:

**Comment by DataM0del** [comment]
We should just convert this to a Svelte Kit project, it has built-in routing support

**Comment by 1zun4** [comment]
Upgrading to Svelte 5 would be great.

**Comment by DataM0del** [comment]
Will be a pain though.
I was just going to use `^5.0.0`, but when I used the vite svelte plugin with the latest version, I just didn't want to change it to 5.0.0 for some reason, or at least that change wasn't kept.

**Comment by 1zun4** [comment]
Upgrading to Svelte 5 will require a lot of code to be refactored. @SenkJu has looked into this at some point.

**Comment by SenkJu** [comment]
> We should just convert this to a Svelte Kit project, it has built-in routing support

Doesn't it require a Svelte Kit server? LiquidBounce requires a static build of the UI.

**Comment by DataM0del** [comment]
> > We should just convert this to a Svelte Kit project, it has built-in routing support
> 
> Doesn't it require a Svelte Kit server? LiquidBounce requires a static build of the UI.

Oh. But no one has a svelte 5 router that works with the latest svelte version (they all depend on `^5.0.0`, we're using `5.2.7`)

**Comment by MukjepScarlet** [comment]
Is SvelteKit static adapter suitable for theme?

**Comment by DataM0del** [comment]
> Is SvelteKit static adapter suitable for theme?

I think so.

**Comment by DataM0del** [comment]
While I would like to rewrite it to use SvelteKit, this PR is dead and I think it would just be better to have a separate PR for migrating to SvelteKit.

**Comment by DataM0del** [comment]
It compiles now, still not working.
image

**Comment by DataM0del** [comment]
I just rebased to 322547d because the other changes (moving to Svelte 5 & sveltekit) are going to take long and I'd rather my console not be spammed with deprecation warnings than wait 10 eons to finally finish the port over to sveltekit.

**Comment by SenkJu** [comment]
This pull request will likely be superseded by https://github.com/CCBlueX/LiquidBounce/pull/4922. I will close this for now. Thank you for your work. You are welcome to contribute to the other pull request if you are interested.

**Pull request closed without merging**